devtools.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'rspec',        '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-core',   '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-its',    '~> 1.2.0'
-  gem.add_runtime_dependency 'rake',         '~> 10.4.2'
+  gem.add_runtime_dependency 'rake',         '~> 11.1.x'
```

Comment:
I typically go with specifying the "latest known patch release I know is stable, since I tested it". So when at the time I specify a dependency the latest known patch release is `11.1.1` I go for `~> 11.1.1`. Explicitly excluding `11.1.0`.